go.mod

```diff
@@ -45,8 +45,9 @@ require (
 	google.golang.org/protobuf v1.27.1
 )
 
+require github.com/ChainSafe/log15 v1.0.0
```

Comment:
Remove usages of `log15` you have in your tests by replacing with `github.com/ChainSafe/gossamer/internal/log` and then re-run `go mod tidy`

Reply:
This is still there, please re-run `go mod tidy` once you got rid of `log15` in your tests.